src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,97 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import javax.validation.constraints.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+import wolox.training.exceptions.BookNonOwnedException;
+
+@Entity
+public class Users {
```

Comment:
I suggest to use the name `User` in order to maintain coherence throughout the code. Same with UsersRepository and UsersController classes :) 

As a good practice, since classes represent single entities they should be named with singular nouns. If your concern is table names and persistence, you can use the annotation `@Entity(name="Users")`